filters/openpolicyagent/opaauthorizerequest/opaauthorizerequest_test.go

```diff
@@ -302,3 +309,174 @@ func isHeadersAbsent(t *testing.T, unwantedHeaders http.Header, headers http.Hea
 	}
 	return true
 }
+
+const (
+	tokenExp = 2 * time.Hour
+
+	certPath = "../../../skptesting/cert.pem"
+	keyPath  = "../../../skptesting/key.pem"
+)
+
+func BenchmarkAuthorizeRequest(b *testing.B) {
+	b.Run("authorize-request-minimal", func(b *testing.B) {
+		opaControlPlane := opasdktest.MustNewServer(
+			opasdktest.MockBundle("/bundles/somebundle.tar.gz", map[string]string{
+				"main.rego": `
+					package envoy.authz
+
+					default allow = false
+
+					allow {
+						input.parsed_path = [ "allow" ]
+					}
+				`,
+			}),
+		)
+
+		f, err := createOpaFilter(opaControlPlane)
+		assert.NoError(b, err)
+
+		url, err := url.Parse("http://somehost/somepath")
```

Comment:
It could be skipper.test or opa.test, see https://www.rfc-editor.org/rfc/rfc2606.html#section-3

Reply:
Good catch. Changed...